packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -105,11 +105,14 @@ const Route = ({
         userId: form.userId,
       },
     });
-    if (!eventTypeValidInContext) {
-      return;
-    }
+
     group.eventTypes.forEach((eventType) => {
       const uniqueSlug = `${group.profile.slug}/${eventType.slug}`;
+      const isLegacy = isRouter(route) ? false : uniqueSlug === route.action.value;
+
+      if (!isLegacy && !eventTypeValidInContext) {
```

Comment:
If it's a legacy case where an existing team event might be selected, let that team event pass through.